app/assets/javascripts/controllers/container_dashboard/util/charts-mixin.js

```diff
@@ -1,6 +1,16 @@
 angular.module('miq.util').factory('chartsMixin', function() {
   'use strict';
 
+  var hourlyTimeTooltip = function(d) {
+    var theMoment = moment(d[0].x);
+    return _.template('<table class="c3-tooltip">' +
```

Comment:
The tooltips for donuts and sparkline charts are coming from C3 charts package. We can modify the content but as of now are not able to modify the tooltip styling to match bootstrap tooltips. Feel free to add an issue to Patternfly / Angular-Patternfly for this.

The Utilization Bar charts are not C3 based so those have the default bootstrap styling.